packages/utilities/adaptive-ui/README.md

```diff
@@ -0,0 +1,49 @@
+# Adaptive UI
+
+Adaptive UI is a library for building highly-consistent design systems based around your visual decisions instead of a complex map of color swatches and tokens based on t-shirt sizes.
```

Comment:
```suggestion
Adaptive UI is a library for building highly-consistent design systems based around your visual decisions.
```

We might want to adjust the language here :)

Reply:
I think this _is_ representative of the current world of design tokens. I'd like to convey it somehow, and wasn't trying to be too tongue-in-cheek, but just realistically accurate. I'll try something a little more descriptive.